### PR TITLE
Adding Scope: Code or Comment for Live templates

### PR DESCRIPTION
**Pull request by QuentinRa**
Hello, I added scopes for live templates (code/comment). Every live template is now disabled inside comments, so that we can write if/while/... without getting weird suggestions.

Inside the settings of our Live templates, we have

Screenshot_3

### CODE REVIEW

File: src/com/reason/ide/template/OCamlContentType.java [hunk 1]
```diff
@@ -13,9 +16,51 @@ protected OCamlContentType() {
...
+    }
+    // Comments only
+    private static final class OCamlCommentTemplates extends ScopeTemplates {
+        public OCamlCommentTemplates() {
+            super("Comment", true);
+        }
+    }
+
+    private static class ScopeTemplates extends OCamlContentType {
+        private final boolean comments;
```
Comment (giraud) [inline src/com/reason/ide/template/OCamlContentType.java:43]: code style is using `my` prefix for class variables and not using `this.`

**Comment by giraud** [comment]
also, when I look at other contexts (like java), they are more precise in the context names: maybe it should be `expression` instead of `code` (too broad ?)

File: src/com/reason/ide/template/OCamlContentType.java (hunk 1 above)
Comment (QuentinRa) [reply src/com/reason/ide/template/OCamlContentType.java:43]: Oh! I wondered why I had a "my" before the name! I will fix it, thanks!

**Comment by QuentinRa** [comment]
😯. Indeed, this would be better, thanks!

**Pull request merged**